app/views/home/site_map.html.erb

```diff
@@ -33,8 +33,9 @@
 		    <li><%= link_to t('.my_collections', :default => "My Collections and Challenges"), user_collections_path(current_user) %></li>
 		    <li><%= link_to t('.my_inbox', :default => "My Inbox"), user_inbox_path(current_user) %></li>
 		    <% if current_user.preference.history_enabled? %>
-		      <li><%= link_to t('.my_history', :default =>"My History"), user_readings_path(current_user) %></li>
+		      <li><%= link_to t('.my_history', :default =>"My Works History"), user_readings_path(current_user) %></li>
 		    <% end %>
+                    <li><%= link_to t('.my_kudos_history', :default =>"My Kudos History"), user_kudos_path(current_user) %></li>
```

Comment:
Turning reading history off means it doesn't track what you view. Turning kudos history off won't stop you leaving kudos, it will just stop you having a handy list. So I'd say there's no useful meaning to turning kudos history off - it won't make anything more secure or private. So I'd leave it as-is.